packages/bootstrap/scss/stepper/_variables.scss

```
@@ -1,87 +1,224 @@
+// The font family of the Stepper.
+/// @group stepper
 $kendo-stepper-font-family: $kendo-font-family !default;
 
+/// The background color of the Stepper.
+/// @group stepper
 $kendo-stepper-bg: null !default;
+/// The text color of the Stepper.
+/// @group stepper
 $kendo-stepper-text: $kendo-component-text !default;
+/// The border color of the Stepper.
+/// @group stepper
 $kendo-stepper-border: null !default;
 
+/// The width of the Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-width: 28px !default;
+/// The height of the Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-height: $kendo-stepper-indicator-width !default;
+/// The border radius of the Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-border-radius: 50% !default;
+/// The border width of the Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-border-width: 1px !default;
 
+/// The border width of the focused Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-focus-border-width: 1px !default;
+/// The size of the focused Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-focus-size: 2px !default;
+/// The offset of the Stepper focused indicator.
+/// @group stepper
 $kendo-stepper-indicator-focus-offset: 3px !default;
+/// The calculated offset of the focused Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-focus-calc-offset: calc( #{$kendo-stepper-indicator-focus-border-width} + #{$kendo-stepper-indicator-focus-offset} ) !default;
 
+/// The background color of the Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-bg: $kendo-component-bg !default;
+/// The text color of the Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-text: $kendo-component-text !default;
+/// The border color of the Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-border: $kendo-component-border !default;
 
+/// The background color of the hovered Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-hover-bg: if($kendo-enable-color-system, k-color( base-hover ), k-try-shade( $kendo-stepper-indicator-bg )) !default;
+/// The text color of the hovered Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-hover-text: null !default;
+/// The border color of the hovered Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-hover-border: null !default;
 
+/// The background color of the disabled Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-disabled-bg: null !default;
+/// The text color of the disabled Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-disabled-text: if($kendo-enable-color-system, rgba( k-color( on-app-surface, true ), .65 ), $kendo-disabled-text) !default;
+/// The border color of the disabled Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-disabled-border: null !default;
 
+/// The background color of the Stepper done indicator.
```

Adding the `'s` for clarity here.
```suggestion
/// The background color of the Stepper's done indicator.
```